proxy/util/cloudsqlutil.go

```diff
@@ -44,3 +47,25 @@ func SplitName(instance string) (project, region, name string) {
 		return spl[0], spl[1], spl[2]
 	}
 }
+
+// versionString indiciates the version of the proxy currently in use.
+//
+//go:embed version.txt
+var versionString string
+
+// metadataString indiciates additional build or distribution metadata.
+var metadataString = ""
+
+// semanticVersion returns the version of the proxy in a semver format.
+func SemanticVersion() string {
+	v := strings.TrimSpace(versionString)
+	if metadataString != "" {
+		v += "+" + metadataString
+	}
+	return v
+}
+
+// userAgentFromVersionString returns an appropriate user agent string for identifying this proxy process.
```

Comment:
Let's try to wrap at 80 characters here.

Reply:
Good catch! Fixed.